src/main/resources/db/changelog/v.0.1/db.changelog-users.yaml

```diff
@@ -0,0 +1,58 @@
+databaseChangeLog:
+  - changeSet:
+      id: 10-add-user-role-enum
+      author: elizaveta_dubogryz
+      dbms: postgresql
+      sql: CREATE TYPE USER_ROLE as ENUM ('USER', 'ADMIN');
+  - changeSet:
+      id: 11-create-users-table
+      author: elizaveta_dubogryz
+      changes:
+        - createTable:
+            tableName: users
+            columns:
+              - column:
+                  name: id
+                  type: UUID
+                  constraints:
+                    primaryKey: true
+              - column:
+                  name: login
+                  type: varchar(50)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: password
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: first_name
+                  type: varchar(50)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: last_name
+                  type: varchar(50)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: email
+                  type: varchar(50)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: phone
```

Comment:
Maybe, put here 
constraints:             
   nullable: true

Also about email. Some people are not sure to share their personality...

Reply:
all employees have corporate email